geniza/annotations/views.py

```diff
@@ -132,7 +132,9 @@ def get(self, request, *args, **kwargs):
         # TODO: Convert this to list when > 2 options
         # implement minimal search by uri
         # implement something similar to SAS search by uri
-        annotations = self.get_queryset()
+
+        # sort by schema:position if available
+        annotations = self.get_queryset().order_by("content__schema:position")
```

Comment:
whoa, does this syntax work?! cool.

Please add a quick test to confirm the sorting; and can we get a secondary sort by creation time? (this is the default sort in the model, so we may get it for "free", but would be good to confirm)

Reply:
Based on the test I wrote it looks like we do, should I still add it to the `order_by`?

Reply:
hmm, doesn't hurt to be explicit I guess. defer to you